### DISCUSSION TITLE
Table组件的搜索框能和搜索框组件一样获取智能提示嘛

### DISCUSSION THREAD

**Discussion by FasterHigherStronger** [dotnetcore/BootstrapBlazor#3856] [Q&A]
![image](https://github.com/user-attachments/assets/63fdefc0-0317-43d7-88a7-67492c5abc6f)很需要这个功能显得更智能一些

**Comment by ArgoZhang** [comment]
@FasterHigherStronger 感谢提交 issue 本页面右上角帮忙点一下 star ，`这个功能` 是那个功能？自动清空搜索不是做了吗？

**Reply by FasterHigherStronger** [reply]
是自动获得焦点那个，不知道table组件自带的搜索框输入后能不能和search组件一样获取智能提示，我去看了文档Table组件的搜索好像并没有这个功能...

**Reply by FasterHigherStronger** [reply]
PS：感谢作者大大回复，star已点
